Makefile

```diff
@@ -29,13 +28,13 @@ lint: ## Lint the code
 	poetry run ruff format --check && \
 	poetry run ruff check && \
 	poetry run mypy .
-	npx prettier --check .
+	docker run --rm -v $(shell pwd):/work tmknom/prettier:3.2.5 --check ./*.md ./docs ./.github/workflows/* ./hushline
```

Comment:
This can be done with `docker compose run --rm ` on the dev container if we install NPM there. Which might be better that pulling in some other container? Idk :shrug:

Reply:
I'd rather not include an extra runtime in our application container if it's only used during development.

Reply:
We have split dev/prod containers, so adding that to the dev container shouldn't conflict with your wishes. Not that we have to add it, but was just sayin'